huaweicloud/services/servicestage/data_source_huaweicloud_servicestagev3_components.go

```diff
@@ -172,16 +189,17 @@ func DataSourceV3Components() *schema.Resource {
 	}
 }
 
-func queryV3Components(client *golangsdk.ServiceClient, appId string) ([]interface{}, error) {
+func listV3Components(client *golangsdk.ServiceClient) ([]interface{}, error) {
 	var (
-		httpUrl = "v3/{project_id}/cas/applications/{application_id}/components?limit=100"
+		httpUrl = "v3/{project_id}/cas/components?limit={limit}"
```

Comment:
```suggestion
		httpUrl = "v3/{project_id}/cas/components?limit=100"
```

Reply:
two places are using it (to reduce one page request if the last page size less than limit value), local vairbale is better.